src/Css.re

```diff
@@ -812,8 +812,45 @@ let gridAutoDirectionToJs =
 let gridAutoFlow = direction =>
   d("gridAutoFlow", gridAutoDirectionToJs(direction));
 
+type repeatValue = [ | `autoFill | `autoFit | `n(int)];
```

Comment:
You're using `n for absolute number. I used `num for the same thing in font type. 
I think that for consistency and readability, we should use `num in both

Reply:
Swapped over to using `num. I couldn't find fontType though. That may have been reworked as part of the changes around #91.